<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VideoDialogClass</class>
 <widget class="QDialog" name="VideoDialogClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>979</width>
    <height>780</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="VideoWidget" name="videoWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>320</width>
       <height>240</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="showControlsBox">
     <property name="text">
      <string>Show controls</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="2">
      <widget class="QSlider" name="shutterSlider">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::NoTicks</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="fpsLabel">
       <property name="text">
        <string>FPS:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="9">
      <widget class="QSpinBox" name="balanceBlueSpinBox"/>
     </item>
     <item row="2" column="2">
      <widget class="QSlider" name="jpegQualitySlider">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::NoTicks</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QSpinBox" name="jpegQualitySpinBox"/>
     </item>
     <item row="0" column="9">
      <widget class="QSpinBox" name="balanceRedSpinBox"/>
     </item>
     <item row="1" column="4">
      <widget class="QLabel" name="labelDb">
       <property name="text">
        <string>dB</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QSpinBox" name="gainSpinBox"/>
     </item>
     <item row="1" column="8">
      <widget class="QSlider" name="balanceBlueSlider">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::NoTicks</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="shutterLabel">
       <property name="text">
        <string>Shutter:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QLabel" name="labelMs">
       <property name="text">
        <string>ms</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="gainLabel">
       <property name="text">
        <string>Gain:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="6">
      <widget class="QLabel" name="wbLabel">
       <property name="text">
        <string>White balance =&gt; </string>
       </property>
      </widget>
     </item>
     <item row="0" column="7">
      <widget class="QLabel" name="balanceRedLabel">
       <property name="text">
        <string>Red:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="8">
      <widget class="QSlider" name="balanceRedSlider">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::NoTicks</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="7">
      <widget class="QLabel" name="balanceBlueLabel">
       <property name="text">
        <string>Blue:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="jpegQualityLabel">
       <property name="text">
        <string>JPEG quality:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="3">
      <widget class="QSpinBox" name="shutterSpinBox"/>
     </item>
     <item row="1" column="2">
      <widget class="QSlider" name="gainSlider">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::NoTicks</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLabel" name="dataRateLabel">
       <property name="text">
        <string>N/A</string>
       </property>
      </widget>
     </item>
     <item row="3" column="6">
      <widget class="QLabel" name="imageSizeLabel">
       <property name="text">
        <string>W: N/A H: N/A</string>
       </property>
      </widget>
     </item>
     <item row="3" column="8">
      <widget class="QLabel" name="imageOffsetLabel">
       <property name="text">
        <string>Offset: (x=N/A, y=N/A)</string>
       </property>
      </widget>
     </item>
     <item row="3" column="5">
      <widget class="QLabel" name="triggerLabel">
       <property name="text">
        <string>Trigger lock:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>VideoWidget</class>
   <extends>QWidget</extends>
   <header>videowidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>shutterSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>VideoDialogClass</receiver>
   <slot>onShutterChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>389</x>
     <y>759</y>
    </hint>
    <hint type="destinationlabel">
     <x>489</x>
     <y>389</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>gainSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>VideoDialogClass</receiver>
   <slot>onGainChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>863</x>
     <y>759</y>
    </hint>
    <hint type="destinationlabel">
     <x>489</x>
     <y>389</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>balanceRedSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>VideoDialogClass</receiver>
   <slot>onBalanceRedChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>875</x>
     <y>720</y>
    </hint>
    <hint type="destinationlabel">
     <x>489</x>
     <y>389</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>balanceBlueSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>VideoDialogClass</receiver>
   <slot>onBalanceBlueChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>875</x>
     <y>755</y>
    </hint>
    <hint type="destinationlabel">
     <x>489</x>
     <y>389</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showControlsBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>VideoDialogClass</receiver>
   <slot>onShowControlsToggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>489</x>
     <y>702</y>
    </hint>
    <hint type="destinationlabel">
     <x>489</x>
     <y>389</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>shutterSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>shutterSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>205</x>
     <y>678</y>
    </hint>
    <hint type="destinationlabel">
     <x>335</x>
     <y>678</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>shutterSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>shutterSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>335</x>
     <y>678</y>
    </hint>
    <hint type="destinationlabel">
     <x>205</x>
     <y>678</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>shutterSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>VideoDialogClass</receiver>
   <slot>onShutterChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>335</x>
     <y>678</y>
    </hint>
    <hint type="destinationlabel">
     <x>489</x>
     <y>389</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>jpegQualitySlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>jpegQualitySpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>197</x>
     <y>725</y>
    </hint>
    <hint type="destinationlabel">
     <x>318</x>
     <y>726</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>jpegQualitySpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>jpegQualitySlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>318</x>
     <y>726</y>
    </hint>
    <hint type="destinationlabel">
     <x>197</x>
     <y>725</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>jpegQualitySlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>VideoDialogClass</receiver>
   <slot>onJpegQualityChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>197</x>
     <y>725</y>
    </hint>
    <hint type="destinationlabel">
     <x>489</x>
     <y>389</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>jpegQualitySpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>VideoDialogClass</receiver>
   <slot>onJpegQualityChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>318</x>
     <y>726</y>
    </hint>
    <hint type="destinationlabel">
     <x>489</x>
     <y>389</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>gainSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>gainSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>197</x>
     <y>698</y>
    </hint>
    <hint type="destinationlabel">
     <x>318</x>
     <y>699</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>gainSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>gainSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>318</x>
     <y>699</y>
    </hint>
    <hint type="destinationlabel">
     <x>197</x>
     <y>698</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>gainSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>VideoDialogClass</receiver>
   <slot>onGainChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>318</x>
     <y>699</y>
    </hint>
    <hint type="destinationlabel">
     <x>489</x>
     <y>389</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>balanceRedSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>balanceRedSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>825</x>
     <y>665</y>
    </hint>
    <hint type="destinationlabel">
     <x>946</x>
     <y>666</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>balanceRedSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>balanceRedSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>946</x>
     <y>666</y>
    </hint>
    <hint type="destinationlabel">
     <x>825</x>
     <y>665</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>balanceRedSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>VideoDialogClass</receiver>
   <slot>onBalanceRedChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>946</x>
     <y>666</y>
    </hint>
    <hint type="destinationlabel">
     <x>489</x>
     <y>389</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>balanceBlueSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>balanceBlueSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>825</x>
     <y>698</y>
    </hint>
    <hint type="destinationlabel">
     <x>946</x>
     <y>699</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>balanceBlueSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>balanceBlueSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>946</x>
     <y>699</y>
    </hint>
    <hint type="destinationlabel">
     <x>825</x>
     <y>698</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>balanceBlueSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>VideoDialogClass</receiver>
   <slot>onBalanceBlueChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>946</x>
     <y>699</y>
    </hint>
    <hint type="destinationlabel">
     <x>489</x>
     <y>389</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onTriggerLockToggled(bool)</slot>
  <slot>onShutterChanged(int)</slot>
  <slot>onGainChanged(int)</slot>
  <slot>onBalanceRedChanged(int)</slot>
  <slot>onBalanceBlueChanged(int)</slot>
  <slot>onShowControlsToggled(bool)</slot>
  <slot>onJpegQualityChanged(int)</slot>
 </slots>
</ui>
